cogs/slowmode.py

```diff
@@ -21,14 +41,57 @@ def __init__(self, client):
     @commands.has_any_role(MOD_ROLE, HELPER_ROLE)
     async def slowmode(self, ctx: commands.Context, interval_str: str):
         '''Activate slowmode in help channels'''
-        # TODO: check against channel ID instead of name
         interval = util.get_id_from_tag(interval_str)
-        if isinstance(ctx.channel, discord.TextChannel) and \
-                ctx.channel.name in ['kb-help', 'kb-buying-advice']:
+        if isinstance(ctx.channel, discord.TextChannel) and ctx.channel.id in CHANNELS:
             await ctx.channel.edit(slowmode_delay=interval)
             await ctx.channel.send("Slow it down!")
 
 
+    def get_delay(self, message_count):
+        message_limits = sorted(time_configs, reverse=True)
+        for limit in message_limits:
+            if message_count >= limit:
+                return time_configs[limit]
+        return 0
+
+
```

Comment:
usually I do one line per function when they're between classes, i'm not sure what you get back from running `make lint` though

Reply:
> > run through `make flake8` and `make pylint` (mypy is mad at a bunch of stuff I still need to fix that has its own branch I've been too lazy, ignore that/run only on cogs/slowmode.py)
>
> this is through docker right? if so, i cant run docker on my machine due to it requiring Hyper-V/WSLv2.